### PR TITLE
CMake DDR: Ignore header include guards

### PR DESCRIPTION
**Pull request by dnakamura**
Signed-off-by: Devin Nakamura <devinn@ca.ibm.com>

### CODE REVIEW

**Comment by youngar** [comment]
LGTM

File: cmake/modules/ddr/cmake_ddr.awk [hunk 1]
```diff
@@ -78,9 +78,11 @@ function write_flag_defined(macro_name) {
 }
 
 function define_flag_macro(macro_name) {
-	# TODO if not include guard
-	write_flag_macro(macro_name);
-	write_flag_defined(macro_name);
+	# Don't process any include guard macros
```
Comment (charliegracie) [inline cmake/modules/ddr/cmake_ddr.awk]: I assume the old makefiles were doing something similar?

**Comment by charliegracie** [comment]
@genie-omr build all

File: cmake/modules/ddr/cmake_ddr.awk (hunk 1 above)
Comment (dnakamura) [reply cmake/modules/ddr/cmake_ddr.awk]: yes
https://github.com/eclipse/omr/blob/2d1e78503606e86fc9751846a252380da8d2e9fe/ddr/tools/getmacros#L116

**Pull request merged**